code/src/components/SecondQuestion.js

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+
+const SecondQuestion =({flavourInput, onFlavourInput, onStepChange}) => {
+return (
+
+<div className="question-box">
+<p className="intro-text">Which macaron <em>flavour</em> do you prefer?</p>
+    <form className="radio-form">
+
+    
+<label className="flavours">
+<input
+type="radio"
+value="Chocolate"
+onChange={onFlavourInput}
+checked={flavourInput === "Chocolate"}
+/>
+<span className="radio-input">Chocolate</span>
+</label>
+
+<label className="flavours">
+<input
+type="radio"
+value="Vanilla"
+onChange={onFlavourInput}
+checked={flavourInput === "Vanilla"}
+/>
+<span className="radio-input">Vanilla</span>
+</label>
+
```

Comment:
There is another way u can write this code, check out map function, makes it alot easyer :)